### PR TITLE
Hunter alerts

### PR DESCRIPTION
**Pull request by stuartgiles**
#### What does this PR do? (please provide any background)

* Tweak state colours (we'll continue to support states in the short term)
* Remove the branded icon state colors for the old hx brand
* Margins and paddingdings, especially items that follow a close button
* Alert bg interit so it works when in an inverted parent
* Alert links text decoration should match default text links
* Alert links border bottom should match default text links
* Alert links color should match default text links
* Package version bump

#### How can this be tested?
`cd pattern-library && npm link ../brands/brands/hunter`
View our Alerts page - http://localhost:8687/components/alerts/index.html

#### Any tech debt?
In future we may follow the designs for the new brand more closely and make all alerts the primary colour, but in the short term we need to continue to support alert states for some of our apps. 

#### Screenshots / Screencast
screen shot 2017-02-16 at 18 04 45

#### What gif best describes how you feel about this work?

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [x] :+1:
- [x] UXUI Team :+1:
- [x] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---

### CODE REVIEW

File: brands/hunter/src/less/alerts.less [hunk 1]
```diff
@@ -4,8 +4,29 @@
...
+  margin: 0;
+
+  * + & {
+    margin-top: ceil((@font-size-default * 2)) // ~30px
+  }
+
+  > p,
+  > ul {
+    margin: 0;
+  }
```
Comment (stuartgiles) [inline brands/hunter/src/less/alerts.less]: My mistake, this is too specific. Should target all `<p>` and `<ul>` that are direct children of an `.alert`.

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by jodiedoubleday** [comment]
:+1:

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Pull request merged**